draft-ietf-rtcweb-jsep.xml

```diff
@@ -1885,15 +1885,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           session-level attributes. The process here is identical to that
           indicated in the Initial Offers section above.</t>
 
-	  <t>The next step is to generate lip sync groups as defined
-	  in <xref target="RFC5888"/>, Section 7.  For each
-	  MediaStream with more than one MediaStreamTrack, a group
-	  of type "LS" MUST be added that contains the mid values for
-	  each MediaStreamTrack in that MediaStream. In some cases
-	  this may result in adding a mid to a given LS group that
-	  was not in that LS group in the associated offer. Although
-	  this is not allowed by <xref target="RFC5888"/>, it is
-	  allowed when implementing this specification. </t>
+	  <t>The next step is to generate session-level lip sync
+	  groups as defined in <xref target="RFC5888"/>, Section
+	  7. All Tracks in the same MediaStream need to be placed in
+	  the same "LS" group as long as theses tracks were in the
+	  offered to by synchronized in the offer.
+	  RFC5888 only allows the groups in the
+	  answer to be a subset of what was in the offer. If there are
+	  tracks that the same MediaStream that can not be added to
+	  the "LS" group, once the state moves back to "stable" the
```

Comment:
This seems nearly right to me. Suggest:

If there are tracks that are placed in the answer, but not within a suitable LS group, or tracks that cannot be added to the answer because of a lack of suitable media sections, once the state moves back to "stable" the browser SHOULD initiate another offer/answer.....